DS13/code/modules/mentor/follow.dm

```diff
@@ -0,0 +1,26 @@
+/client/proc/mentor_follow(mob/living/M)
+	if(!is_mentor())
+		return
+	if(isnull(M))
+		return
+	if(!ismob(usr))
+		return
+	mentor_datum.following = M
+	usr.reset_perspective(M)
+	verbs += /client/proc/mentor_unfollow
+	to_chat(GLOB.admins, "<span class='mentor'><span class='prefix'>MENTOR:</span> <EM>[key_name(usr)]</EM> is now following <EM>[key_name(M)]</span>")
+	to_chat(usr, "<span class='info'>Click the \"Stop Following\" button in the Mentor tab to stop following [key_name(M)].</span>")
+	log_mentor("[key_name(usr)] began following [key_name(M)]")
+
+/client/proc/mentor_unfollow()
```

Comment:
Does this show up on rightclick? OR is it like FLW on tickets?

Reply:
Like FLW on tickets